orchestrator/cosmos_gravity/src/send.rs

```diff
@@ -120,20 +120,28 @@ pub async fn send_request_batch_tx(
         denom,
     };
     let msg = Msg::new("/gravity.v1.MsgRequestBatchTx", msg_request_batch);
-    send_messages(contact, cosmos_key, fee, vec![msg]).await
+    send_messages(contact, cosmos_key, gas_price, vec![msg]).await
 }
 
 pub async fn send_messages(
     contact: &Contact,
     cosmos_key: CosmosPrivateKey,
-    fee: Coin,
+    gas_price: Coin,
```

Comment:
Does Rust/gorc have a notion of `DecCoin`? This is what operators would be providing. e.g. `0.001stake`.

Reply:
Good question -- checking deep_space.

Reply:
So yeah, it'll be something like: `int(ceil(gas_price.amount * float64(gas_limit)))`

Reply:
No deep_space::DecCoin. There is a Dec though & we could use that w/ a slightly different config syntax (closer to what we saw in the beginning):

```toml
[cosmos]
gas_price = { amount = "0.001", denom = "stake" }
```

Will explore it a bit.

Reply:
Yup! That makes sense to me!